cypress/integration/movietest.spec.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-undef */
+
+describe('movie-page', () => {
+    
+    beforeEach(() => {
+      cy.visit('http://localhost:3000/')
```

Comment:
Check this information https://docs.cypress.io/guides/references/best-practices#Setting-a-global-baseUrl is a good practice to set the base URL in the cypress config file